tests/unit/single_table/test_base.py

```diff
@@ -880,7 +880,7 @@ def test__sample_rows_without_conditions(self):
         })
         instance = Mock()
         instance._random_state_set = False
-        instance._data_processor._dtypes = pd.Series()
+        instance._data_processor._dtypes = pd.Series
```

Comment:
pd.Series is a function right? I don't think this makes sense. We should set the dtypes to a series of actual dtype values. If the values don't actually matter for the test then we can set to an empty series of dtype=object

Reply:
Hi @amontanez24. Actually I removed those lines because they were useless in the tests. All is working know and I no longer see the warning in any tests. Mainly, that was the lines in `tests/unit/constraint/test_base.py` that were raising 100 warnings each. [50afae0](https://github.com/sdv-dev/SDV/pull/1373/commits/50afae002e2b55fc69fc045a29fdb24ced2ca04c)